docs/python/psycopg2/README.md

```diff
@@ -11,7 +11,6 @@
     - [CommenterCursorFactory](#commentercursorfactory)
       - [with_openCensus=True](#with_opencensustrue)
     - [Expected fields](#expected-fields)
-      - [Default options with flask](#default-options-with-flask)
```

Comment:
This section is also present in the doc.

Reply:
Thanks for catching this!